src/test/resources/pages/instructorFeedbackResultsAddComment.html

```diff
@@ -835,7 +835,7 @@
                                              <span class="text-preserve-space">
                                                 What is your extra feature? 
                                                 <span style=" white-space: normal;">
-                                                   <a class="color_gray" data-less="[less]" data-more="[more]" href="javascript:;" id="{*}" onclick="toggleAdditionalQuestionInfo('7-giver-1-recipient-1')">
+                                                   <a class="color_gray" data-less="[less]" data-more="[more]" href="javascript:;" id="{***}" onclick="toggleAdditionalQuestionInfo('7-giver-1-recipient-1')">
```

Comment:
The ID is in form of `questionAdditionalInfoButton-${questionNumber}-${additionalInfoId}`, which does match the regex for regkey. Erroneously replaced.